omod_1.x/src/main/webapp/cashierMessageProperties.jsp

```diff
@@ -0,0 +1,27 @@
+<%@page contentType="text/javascript" pageEncoding="UTF-8"%>
```

Comment:
Have you tested and confirmed that this page will be cached by the browser and not reloaded every time?